docs/docsite/rst/desired_state_schema.rst

```diff
@@ -0,0 +1,61 @@
+.. _ansible_collections.cloud.pravic.docsite.desired_state:
+
+YAML specification for defining desired resource state
+======================================================
+
+The users who intend to use the `cloud.pravic Collection <https://github.com/ansible-collections/pravic>`_ to provision/deprovision Cloud resources will need to use this YAML specification for defining desired resource state:
+
+```yaml
+  ResourceGroups:
+  description: Array of resource groups.
+  type: array
+  items:
+    type: object
+    properties:
+      ResourceGroupName:
+        description: The unique identifier for a group of resources. ResourceGroupName should not include any spaces.
+        type: string
+      Resources:
+        description: Dictionary of resources belonging to ResourceGroupName.
+        type: object
+        properties:
+          name:
+            description: The unique identifier for a resource. It is a hash map with the name as the key."
```

Comment:
```suggestion
            description: A locally unique identifier for a resource. It is a hash map with the name as the key."
```
I would clarify here that this is locally unique and shouldn't be confused with the cloud provider's unique identifier for the resource.